api/src/main/java/io/strimzi/api/kafka/model/kafka/entityoperator/EntityTopicOperatorSpec.java

```diff
@@ -79,16 +81,30 @@ public void setImage(String image) {
         this.image = image;
     }
 
-    @Description("Interval between periodic reconciliations.")
+    @Description("Interval between periodic reconciliations in seconds.")
     @Minimum(0)
     @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-    public int getReconciliationIntervalSeconds() {
+    @DeprecatedProperty(description = "Use reconciliationIntervalMs converting the value to milliseconds.")
+    @PresentInVersions("v1alpha1-v1beta2")
+    @Deprecated
+    public long getReconciliationIntervalSeconds() {
```

Comment:
Have you tried the OneOf annotation? I do not remember out of my head if it makes one of the fields required or not. If it keeps both of them as optional, we should use it here.

Reply:
Yes, and it makes it required.

Reply:
Hmm, then we can not use it. Shame. But thanks for checking!